skipper.go

```diff
@@ -307,7 +308,7 @@ func Run(o Options) error {
 	}
 
 	// include bundeled custom predicates
-	o.CustomPredicates = append(o.CustomPredicates, source.New())
+	o.CustomPredicates = append(o.CustomPredicates, source.New(), interval.NewBetweenPredicate(), interval.NewBeforePredicate(), interval.NewAfterPredicate())
```

Comment:
we didn't define strict rules on the line lengths, but this feels a bit too long. Can we cut it somehow?